gpMgmt/test/behave/mgmt_utils/gpstop.feature

```diff
@@ -38,3 +38,10 @@ Feature: gpstop behave tests
           And gpstop should print "invalid_host is unreachable.Skipping cleaning shared memory." to stdout
           And gpstop should return a return code of 0
           And the standby host is made reachable
+
+	@demo_cluster
+    Scenario: gpstop exits without a warning
```

Comment:
Provide more verbose scenery description. There are may be logged dozens of different warning during cluster stop, but you check some exact.

Reply:
Added more verbose description